cypress/integration/pages/testsForAllPages.js

```diff
@@ -365,6 +365,20 @@ export const testsThatFollowSmokeTestConfigforAllPages = ({
           .should('have.attr', 'href', '#content');
       });
 
+      if (variant === 'default') {
```

Comment:
This will run this script link cypress test for every single page in our configuration. Following our discussions in our test retrospective session, we should have integration tests for the Header container to ensure the Script link is applied correctly on relevant services and remove this Cypress test which will run on every service for every pagetype in the test suite.